sdk/src/main/java/com/hedera/hashgraph/sdk/TokenMintTransaction.java

```diff
@@ -51,6 +54,28 @@ public long getAmount() {
         return builder.getAmount();
     }
 
+    public TokenMintTransaction addMetadata(byte[] metadata) {
+        requireNotFrozen();
+        builder.addMetadata(ByteString.copyFrom(metadata));
+        return this;
+    }
+
+    public TokenMintTransaction addMetadatas(List<byte[]> metadatas) {
```

Comment:
I think you mean to use `setMetadata()` here instead of `addMetadatas()`. _imo_ This should completely override the list as well.